internal/topic/topic-operation.go

```diff
@@ -170,26 +170,26 @@ func (operation *Operation) DescribeTopic(topic string, flags DescribeTopicFlags
 
 func (operation *Operation) printTopic(topic Topic, flags DescribeTopicFlags) error {
 
-	if flags.PrintConfigs {
-		if flags.OutputFormat == "json" || flags.OutputFormat == "yaml" {
-			topic.Configs = nil
```

Comment:
actually I don't see why this was intentionally set to `nil` here, so your change makes sense :+1: